openassessment/tests/data/db_fixtures/scored.json

```diff
@@ -978,23 +978,5 @@
       "student_item": 1,
       "points_possible": 20
     }
-  },
-  {
-    "pk": 1,
-    "model": "submissions.scoresummary",
-    "fields": {
-      "highest": 1,
-      "student_item": 2,
-      "latest": 1
-    }
-  },
-  {
-    "pk": 2,
-    "model": "submissions.scoresummary",
-    "fields": {
-      "highest": 2,
-      "student_item": 1,
-      "latest": 2
-    }
```

Comment:
@macdiesel @muhammad-ammar Signals are now being called during loading of fixtures so these models are automatically created because of this signal handler: https://github.com/edx/edx-submissions/blob/master/submissions/models.py#L262-L299